great_expectations/core/expectation_suite.py

```diff
@@ -133,6 +134,13 @@ def __init__(  # noqa: PLR0913
     def name(self) -> str:
         return self.expectation_suite_name
 
+    @property
+    def expectations(self) -> list[Expectation]:
+        return [
+            self._build_expectation(expectation_configuration=expectation_configuration)
+            for expectation_configuration in self.expectation_configurations
+        ]
```

Comment:
breaking changes:
- returning Expectations instead of ExpectationConfigurations
- build_expectation validates that ExpectationConfiguration is valid